.changeset/rewrite-changelog.mjs

```diff
@@ -40,13 +40,18 @@ let latestVersion = undefined;
 let stoppedIndex = -1;
 let subHeader = false;
 
+// Get today's date in YYYY-MM-DD format
+const today = new Date();
+const formattedDate = today.toISOString().split('T')[0];
+
 for (const line of currentLines) {
 	stoppedIndex++;
 
 	if (line.startsWith('## ')) {
 		if (!latestVersion) {
 			latestVersion = line.replace('## ', '');
-			newLines.push(line);
+			// Add today's date to the latest version
+			newLines.push(`## ${latestVersion} - ${formattedDate}`);
```

Comment:
[suggestion] Considering a possible gap between a date running this script and an actual release date, it seems more proper to just leave a placeholder. For example:

```suggestion
			newLines.push(`## ${latestVersion} - YYYY-MM-DD ⚠️ Replace this placeholder with an actual release date and remove this warning`);
```

Any thoughts?

Reply:
As we tend to merge a couple of final PRs before release, I think sometimes the date will be correct. Shall we assume it will be and add a note to [our release process](https://stylelint.io/maintainer-guide/releases/#:~:text=Release%20a%20Stylelint) as we already have an action there to check the order of the items within the changelog.